python/tests/conftest.py

```diff
@@ -50,6 +51,13 @@ def assert_message(message_id):
         assert any([message_id in r.message for r in caplog.records])
     return assert_message
 
+@pytest.fixture
+def assert_info(caplog):
```

Comment:
I would have probably just added a 'level=logging.WARN(ING)' parameter to assert_log to avoid duplicated functionality and then create a wrapper if used several times, but if you like current solution more it's OK.